nsxt/policy_utils.go

```diff
@@ -188,6 +188,67 @@ func getResourceIDFromResourcePath(rPath string, rType string) string {
 	return ""
 }
 
+func parseStandardPolicyPath(path string) ([]string, error) {
+	var parents []string
+	segments := strings.Split(path, "/")
+	if len(segments) < 3 {
+		return nil, fmt.Errorf("invalid policy path %s", path)
+	}
+	if segments[0] != "" {
+		return nil, fmt.Errorf("policy path is expected to start with /")
+	}
+	// starting with *infra index
+	idx := 1
+	infraPath := true
+	if segments[1] == "orgs" {
+		if len(segments) < 5 {
+			return nil, fmt.Errorf("invalid multitenant policy path %s", path)
+		}
+
+		// append org and project
+		parents = append(parents, segments[2])
+		parents = append(parents, segments[4])
+		idx = 5
+
+		if len(segments) > 6 && segments[5] == "vpcs" {
+			parents = append(parents, segments[6])
+			idx = 7
+			// vpc paths do not contain infra
+			infraPath = false
+		}
+	}
+	if len(segments) <= idx {
+		return nil, fmt.Errorf("unexpected policy path %s", path)
+	}
+	if infraPath {
+		// continue after infra marker
+		if segments[idx] != "infra" && segments[idx] != "global-infra" {
```

Comment:
I don't remember a case where we can have global-infra for a project-scoped resource.
At the same time, I don't think that a special case to handle only "infra" when in org scope is mandatory